### PR TITLE
Fix a couple of bugs in cleanup

### PR DESCRIPTION
**Pull request by tristanpemble**
Ran into a couple of bugs when cleaning up the tables

### CODE REVIEW

File: src/Lhm/Lhm.php [hunk 1]
```diff
@@ -152,12 +154,12 @@ public static function cleanup($run = false, array $options = [])
                 $adapter->query("DROP TRIGGER IF EXISTS {$trigger}");
             }
 
+            $adapter->query('SET foreign_key_checks=0');
```
Comment (masom) [inline src/Lhm/Lhm.php:157]: Curious why would a LHM table have a foreign key associated to it?

Comment (tristanpemble) [inline src/Lhm/Lhm.php:157]: I was too -- I'm not sure. I will look into it more, but it definitely had foreign keys.

Comment (masom) [inline src/Lhm/Lhm.php:157]: I took a quick look at Soundcloud/lhm and it does not seem to be disabling foreign keys. Would you be able to provide a test case that requires FK checks?

Comment (masom) [inline src/Lhm/Lhm.php:157]: The actual LHM code is: https://github.com/soundcloud/lhm/blob/8d6e8fdfa3b2ca0be1d719d8720759eaec193d65/lib/lhm.rb#L59

Comment (masom) [inline src/Lhm/Lhm.php:157]: @tristanpemble any updates on this?

**Comment by masom** [comment]
@tristanpemble I'll open a separate PR that only changes the `tablestoclean` variable.

**Pull request closed without merging**